Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM public.ecr.aws/unocha/python:3
+
+WORKDIR /srv/hapi
+
+COPY . .
+
+RUN apk add unit unit-python3 && \
+    apk --virtual .build-deps add \
+        git \
```

Comment:
duplicate.
spread them all out one per line, easier to read, although its build deps and get removed afterwards.

Reply:
Added the changes you suggested